readinglist/resource.py

```diff
@@ -254,8 +265,9 @@ def _extract_filters(self, queryparams):
 
     def _extract_sorting(self, queryparams):
         """Extracts filters from QueryString parameters."""
-        specified = queryparams.get('_sort', '').split(',')
+        specified = queryparams.get('_sort', '').split(',')[:1]
```

Comment:
IHMO it is not fair to limit sorting to one field here.

You should take the first sorting parameter into account (and forget the other ones), only if paginating